releases/nightly-build/compose-files/docker-compose-nexus-mongo-arm64.yml

```diff
@@ -148,7 +133,7 @@ services:
       - consul-scripts:/consul/scripts:ro,z
       - /tmp/edgex/secrets:/tmp/edgex/secrets:z
     depends_on:
-      - volume
+      - security-secrets-setup
       - consul
```

Comment:
`consul` dependency here can be remove?

Reply:
No... Note dependency on consul-scripts because edgex-mongo needs health checking scripts.  If anything I should have added vault-worker for the security-enabled compose-files.

Reply:
got it